File: oracle/jruby/delete/oracle_delete.rb
#! /usr/bin/jruby
# -*- coding: utf-8 -*-
#
#	oracle_delete.rb
#
#				Aug/06/2015
#
include Java
import java.sql.DriverManager
import java.lang.System
#
load '/var/www/data_base/common/jruby_common/jruby_sql_manipulate.rb'
#
# ------------------------------------------------------------
puts "*** 開始 ***"
#
id_in = ARGV[0]
#
puts id_in
#
host = "host_oracle"
user = "scott"
password = "tiger"

str_connect = "jdbc:oracl:thin:@" + host + ":1521/xe"

conn = DriverManager.getConnection(str_connect,user,password)
#
sql_delete_proc(conn,id_in)
#
conn.close()
#
puts "*** 終了 ***"
#
# ------------------------------------------------------------

File: oracle/jython/delete/oracle_delete.py
#! /usr/bin/jython
# -*- coding: utf-8 -*-
#
#	oracle_delete.py
#
#				Aug/05/2015
#
# --------------------------------------------------------
import sys
import string
#
import java
from java.sql import DriverManager
#
sys.path.append ('/var/www/data_base/common/jython_common')
#
from jython_rdb_manipulate import display_proc,delete_proc
#
# --------------------------------------------------------
print ("*** 開始 ***")
id_in = sys.argv[1]
print ("%s" % id_in)
#
host = "host_oracle"
user = "scott"
password = "tiger"

str_connect = "jdbc:oracl:thin:@" + host + ":1521/xe"
#
conn = DriverManager.getConnection (str_connect,user,password)
#
delete_proc	(conn,id_in)
#
conn.close ()
print ("*** 終了 ***")
#
# --------------------------------------------------------
